e2e-tests/tests/async-ignore-annotations/02-add-custom-annotations.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  - script: |-
+      set -o errexit
+      set -o xtrace
+
+      source ../../functions
+      mysql_svc=$(get_mysql_service async-ignore-annotations)
+      haproxy_svc=$(get_haproxy_svc async-ignore-annotations)
+      orc_svc=$(get_orc_svc async-ignore-annotations)
+
+      kubectl get svc  $mysql_svc-0 -o yaml -n "${NAMESPACE}" \
+      	| yq eval '.metadata.annotations += {"async-ignore-annotations-mysql": "true", "async-not-ignore-annotations-mysql": "true" }' - \
```

Comment:
Please use better wording here like
'annotations-to-mind/annotations-to-ignore'

Reply:
I'm looking for a better name :) It was painful! I like your suggestion! I will fix it.